src/opnsense/www/js/bootstrap-select.js

```diff
@@ -1022,7 +1022,7 @@
           }
         }
 
-        if (typeof classes !== 'undefined' & '' !== classes) a.className = classes;
+        if (typeof classes !== 'undefined' && '' !== classes) a.className = classes;

```

Comment:
As I said in the initial post: 

> I have opened a pull request upstream for the operator fix, but it will probably take a while before that hits a stable release.